lib/src/transaction_builder.dart

```diff
@@ -90,6 +92,25 @@ class TransactionBuilder {
     return _tx.addOutput(scriptPubKey, value);
   }
 
+  int addNullOutput(dynamic data) {
+    var scriptPubKey;
+    if (data is String) {
+      if (data.length <= network.opreturnSize) {
+        scriptPubKey = bscript.compile([OPS['OP_RETURN'], utf8.encode(data)]);
+      } else {
+        throw new ArgumentError('Too much data, max OP_RETURN size is '+network.opreturnSize.toString());
+      }
+    } else if (data is Uint8List) {
+      scriptPubKey = data;
+    } else {
+      throw new ArgumentError('Invalid data');
+    }
+    if (!_canModifyOutputs()) {
+      throw new ArgumentError('No, this would invalidate signatures');
+    }
+    return _tx.addOutput(scriptPubKey, 0);
```

Comment:
This method is very similar to the `addOutput` method with duplicated code. This duplication should be removed.

Reply:
Fixed.